examples/next-2024/tsconfig.json

```diff
@@ -17,10 +21,21 @@
         "name": "next"
       }
     ],
+    "baseUrl": ".",
     "paths": {
-      "@/*": ["./*"]
+      "@lib/*": ["lib/*"],
+      "@components/header": ["components/header.ts"],
```

Comment:
**question**: Are all these path aliases still in use?

Reply:
Done remove all the path, the ts config is already use in another example so we no longer need this